cmd/daytona/config/const.go

```diff
@@ -122,6 +122,12 @@ func GetPrebuildScopesFromGitProviderId(providerId string) string {
 		fallthrough
 	case "github-enterprise-server":
 		return "admin:repo_hook"
+	case "gitlab":
+		fallthrough
+	case "gitlab-self-managed":
+		return "api,read_repository,write_repository"
+	case "bitbucket":
+		return "webhook"
```

Comment:
Why did you add cases for gitlab here? Please don't add anything not related to the issue you are solving without discussing it first with the maintainers.
Only add bitbucket relevant code, and instead of 'webhook' add 'webhooks' since that's what it's written like when you're creating app password in bitbucket.

Reply:
My bad, sorry about that!

Reply:
Let's add 'webhooks' instead of 'webhook' as I mentioned earlier

Reply:
Just realised you missed this. Accidental approve, sorry